src/style.css

```diff
@@ -1 +1,31 @@
-/* add styles here */
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url('../images/star.svg');
+  background-repeat: no-repeat;
+  background-position: center;
+  height: 16px;
+  width: 16px;
+}
```

Comment:
Don't use tag names for styling (except `html` and `body`). Style elements using `.classes`. In this case, `.stars__star` is already a class, so there's no issue, but be mindful of this rule for future styling.